vector/build.gradle

```diff
@@ -254,6 +254,8 @@ dependencies {
         exclude group: 'com.google.firebase', module: 'firebase-measurement-connector'
     }
 
+    implementation fileTree(dir: 'libs', include: ['*.aar', '*.jar'], exclude: [])
```

Comment:
I prefer to include the jars by specifing its full name rather than a too large wild card. That way you can specify where the file diff_match_patch_current.jar comes from, and how to upgrade the version

Reply:
i finally found a maven repo working